### DISCUSSION TITLE
Valuations.csv results

### DISCUSSION THREAD

**Discussion by SONOFSWAAD** [devfinwiz/Stock_Screeners_Raw#2] [Q&A] [answered]
I had another query. How does your script to determine fair valuations use alone book value and annual sales to arrive at a fair value for the stock? Like does it have some corelation with something else which isn't made public?

**Comment by devfinwiz** [comment, marked as answer]
THe two parameters book value and sales are not connected for my algorithm directly to spot stock's fair value. The annual sales part can be considered similar to the marketcap-annual sales query that i cleared in your previous question. And, for the book value parameter, without complicating it for you in financial terms, a decent stock which is fundamentally sound (based on n parameters not just 1 or 2) always trade over its book value. These two parameters arecombined in the result to apply them as a filter in an attempt to spot better investment opportunities.  Hope this answers your query :)

**Reply by SONOFSWAAD** [reply]
So basically, just to confirm, the two parameters are not connected to their functioning but used together to check if they are together producing better results for the user? Right?

**Reply by devfinwiz** [reply]
Yep, exactly!
